packages/xo-web/src/xo-app/new/network/index.js

```diff
@@ -63,10 +63,13 @@ If and only if the PIF:
 For more info see: https://xapi-project.github.io/xapi/design/tunnelling.html
 */
 const canSupportPrivateNetwork = (pool, pif) =>
-  (pif.isBondMaster || pif.physical || pif.vlan !== -1) &&
-  pif.mode !== 'None' &&
-  !pif.isBondSlave &&
-  pif.$host === pool.master
+  pool !== undefined &&
+  pif !== undefined &&
+    ? false
```

Comment:
In case `pif` and `pool` are defined it will return `false`.
We want the opposite. If `pool` or `pif` is not defined, return `false`.
So:
```js
(pool === undefined || pif === undefined) ? false : ...
```
Or:
```js
pool !== undefined &&
pif !== undefined &&
(pif.isBondMaster || ...)
```

Reply:
Also, if the condition becomes too complicated, you can use early returns to make it more readable:
```js
if (pool === undefined || pif === undefined) {
  return false
}

return ...
```